### DISCUSSION TITLE
Ideas for improvements for the Chat Card

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#2159] [Ideas]
1. Feedback: I would like to optionally turn on feedback for the GenAI responses in the form of a Thumbs Up / Thumbs Down buttons for each chat response that could be new events. As a developer, I would want to know the text and the button clicked (thumbs up or down). I would record the feedback but not make any change to the UI other than I would want the button to look selected. 

2. Disabled: I would like to be able to disable and enable the card. Specifically, when the user asks a question and an LLM generating a response, I do not want the user to ask more questions. Today I can fake this with a `q.client.is_busy` boolean that ignores user inputs when True but... it's messy

**Comment by mturoci** [comment]
> 1

Absolutely. We can fire it as an event.

> 2

We have native global loading spinner. If that's not enough though, I am fine with adding a `disabled` prop for disabling the textbox. 

@mtanco please file feature requests for both of the above.
